packages/frontend/src/components/AppBar/index.tsx

```diff
@@ -79,11 +101,29 @@ export default function AppBar(props: AppBarProps): React.ReactElement {
             colorScheme="secondary"
             target="_blank"
             variant="link"
-            mr={6}
+            mr={4}
           >
             Guide
           </Link>
 
+          <NewsDrawer handleOpen={handleOpen}></NewsDrawer>
+
+          {localLatestTimestamp !== latestNewsTimestamp && (
+            <Icon
+              boxSize={2}
+              ml="-1.2rem"
+              mb="0.8rem"
+              mr="0.5rem"
+              viewBox="0 0 200 200"
+              color="red.500"
+            >
+              <path
+                fill="currentColor"
+                d="M 100, 100 m -75, 0 a 75,75 0 1,0 150,0 a 75,75 0 1,0 -150,0"
+              />
+            </Icon>
+          )}
+
```

Comment:
why put this chunk of code here? this belongs in NewsDrawer component

Reply:
Also, this can be simplified into a box with borderRadius 50%

Reply:
thank you sir